src/Infrastructure/Share/Event/Event.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Share\Event;
+
+use App\Infrastructure\Share\Bus\EventInterface;
+use Broadway\Domain\DomainMessage;
+
+final class Event implements EventInterface
```

Comment:
If it's final I think the interface isn't needed then. I am wrong?

Reply:
`EventInterface` allows it to be added to the events message bus. I want to be consistent with query and command buses.

Reply:
Why not publish a domain message instead this class? The reason is that this class is internal only and it's basically wrapping DomainMessage, that wraps the event. I don't see value in this. Can you give me some arguments about where is needed like that apart of act as a bridge for Broadway lib?

Reply:
I decided to use wrapper class because we shouldn't make applications dependent on external libraries. Their exchange in future can be annoying and time consuming. In fact this repository is template so should be flexible as well.

Reply:
Good point. Agreed